nav2_bt_navigator/behavior_trees/parallel_w_recovery.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+TODO(orduno
+-Provide an explanation
+- A recovery node (e.g. stuck_recovery) returns SUCCESS if it was able to execute all actions.
+  That doesn't mean we fixed the failure, we need to check the condition again.
+-->
+
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <Sequence name="root">
+      <FallbackStar name="check_motion">
+        <Inverter name="is_not_stuck">
+          <IsStuck/>
+        </Inverter>
+        <SequenceStar name="stuck_recovery">
+          <Stop/>
+          <BackUp/>
+          <Spin/>
+        </SequenceStar>
+      </FallbackStar>
+      <SequenceStar name="navigate">
+        <ComputePathToPose endpoints="${endpoints}" path="${path}"/>
+        <Parallel threshold_M="1">
```

Comment:
With the new upstream BT code, "Parallel" is now "ParallelNode" and "threshold_M" is now just "threshold"

Reply:
Fixed